tests/system/test_system.py

```diff
@@ -443,6 +446,58 @@ def test_create_database_pitr_success(self):
             for result in results:
                 self.assertEqual(result[0], retention_period)
 
+    @unittest.skipIf(
+        USE_EMULATOR, "Default leader setting is not supported by the emulator"
+    )
+    def test_create_database_with_default_leader_success(self):
+        pool = BurstyPool(labels={"testcase": "create_database_default_leader"})
+
+        # Create a multi-region instance
+        multi_region_config = "nam3"
+        ALT_INSTANCE_ID = "new" + unique_resource_id("-")
+        config_name = "{}/instanceConfigs/{}".format(
+            Config.CLIENT.project_name, multi_region_config
+        )
+        instance = Config.CLIENT.instance(
+            instance_id=ALT_INSTANCE_ID, configuration_name=config_name
+        )
+        operation = instance.create()
+        self.instances_to_delete.append(instance)
+        operation.result(SPANNER_OPERATION_TIMEOUT_IN_SECONDS)
+
+        temp_db_id = "temp_db" + unique_resource_id("_")
+        default_leader = "us-east4"
+        ddl_statements = [
+            "ALTER DATABASE {}"
+            " SET OPTIONS (default_leader = '{}')".format(temp_db_id, default_leader)
+        ]
+        temp_db = instance.database(
+            temp_db_id, pool=pool, ddl_statements=ddl_statements
+        )
+        operation = temp_db.create()
+        self.to_delete.append(temp_db)
+
+        # We want to make sure the operation completes.
+        operation.result(30)  # raises on failure / timeout.
+
+        database_ids = [database.name for database in instance.list_databases()]
+        self.assertIn(temp_db.name, database_ids)
+
+        temp_db.reload()
+        self.assertEqual(temp_db.default_leader, default_leader)
+
+        with self.assertRaises(exceptions.InvalidArgument):
+            temp_db.create()
```

Comment:
This doesn't make sense. The database already exists. With the above change to instance create, I would move this to a separate test like `test_create_database_with_default_leader_invalid_args`.

Reply:
Thanks for spotting this, I must've accidentally pasted this in. I've deleted it now.